package.json

```diff
@@ -80,6 +80,7 @@
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@cocos/box2d": "1.0.1",
     "@cocos/cannon": "1.2.8",
+    "@webgpu/glslang": "^0.0.15",
```

Comment:
I saw 'glslang' has aready been in the `external/webgpu`, why do we need add this module here?